web/taskmaster/src/app/task.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable, inject } from "@angular/core";
+import { Task } from "./task.model";
+import { HttpClient } from "@angular/common/http";
+import { Observable } from "rxjs";
+
+@Injectable({
+    providedIn: 'root'
+})
+export class TaskService {
+    http = inject(HttpClient)
+
+    getTasks(): Observable<Task[]> {
+         return this.http.get<Task[]>('http://localhost:8001/tasks')
```

Comment:
You can use the CLI to create the env files using `ng generate environments`. This will generate an environment file for development, stage and production. By default the configuration in angular.json will change the one in used by just importing environment.ts into your file. We use this initially for the API route, in this case `http://localhost:8001` 

https://angular.dev/tools/cli/environments#